src/handlers/events/delivery/build/SetStatusOnBuildComplete.ts

```diff
@@ -41,12 +41,11 @@ import { LogInterpretation } from "@atomist/sdm/spi/log/InterpretedLog";
 import { RepoRefResolver } from "@atomist/sdm/spi/repo-ref/RepoRefResolver";
 import {
     BuildStatus,
-    OnBuildComplete,
+    OnBuildComplete, SdmGoalState,
```

Comment:
format imports

Reply:
that can be an autofix, that is not worth our personal attention